platform/viewer/src/appExtensions/GenericViewerCommands/commandsModule.js

```diff
@@ -16,24 +18,84 @@ const actions = {
     newIndex = newIndex < 0 ? maxIndex : newIndex;
 
     store.dispatch(setViewportActive(newIndex));
-  }
+  },
+  setWindowLevelPreset: ({ viewports, preset }) => {
+    const state = store.getState();
+    const { preferences = {} } = state;
+    const { window, level } =
+      preferences.windowLevelData && preferences.windowLevelData[preset];
+
+    if (window && level) {
+      commandsManager.runCommand('setWindowLevel', {
+        viewports,
+        window,
+        level,
+      });
+    }
+  },
 };
 
 const definitions = {
   // Next/Previous active viewport
   incrementActiveViewport: {
     commandFn: actions.updateActiveViewport,
-    storeContexts: ["viewports"],
-    options: { direction: 1 }
+    storeContexts: ['viewports'],
+    options: { direction: 1 },
```

Comment:
Hmm.. We can "cache" updated hotkeys when users change them. Maybe we also allow configs to set "options" to call for hotkeys by default, and on save we can cache these updated options? That way, we're not saving the windowLevelPreset index, and can instead store the width/level as options.

Reply:
I agree with you Danny, I will update this issue (#1447) so we can update hotkeyManager and get this done on a different PR